### ISSUE TITLE
Pass the whole event to process managers

### ISSUE THREAD

**Issue by LostKobrakai** [CargoSense/fable#9]
Currently process managers only receive the event data. This doesn't necessarily include important details like the aggregate_id/…_table, so one can query the changed aggregate. I'd guess giving the callback the complete event would be the way to go, but it would make matching on the event type a bit more noicy.

**Comment by benwilson512** [comment]
Yeah I've gone back and forth about whether I should do this or not, same with the `handler` functions too. It sort of boils down to whether the entire events table row is considered the event, or whether that stuff is "implementation" and only the actual jsonb column is the event. 

All in all however I think preventing people from accessing that information is a losing battle, so we should probably pass in the whole row. If we're going to do so for process managers however we should probably do so for the handler functions as well so that we're consistent.

**Comment by LostKobrakai** [comment]
To me it‘s less the „whole“ event, but mostly getting access to the aggregate. E.g. when I have an important event for an aggregate, where I want to send out a email about the change to the owner I probably need the aggregate. The event data alone might not include anything to get the aggregate though.
